fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -175,6 +175,18 @@
 
     private static final String DRUID_HEADER_SUPPLIER_CLASS = "druid_header_supplier_class";
 
+    /**
+     * Druid filter builder related.
+     */
+    private static final String FILTER_BUILDER_INIT_ERROR = "Unable to create Druid filter builder";
+    private static final String DRUID_FILTER_BUILDER_KEY = "druid_filter_builder";
+    private static final String DEFAULT_DRUID_FILTER_BUILDER =
+            "com.yahoo.bard.webservice.data.filterbuilders.DruidInFilterBuilder";
+    private static final String DRUID_FILTER_BUILDER = SYSTEM_CONFIG.getStringProperty(
```

Comment:
Why are we using configuration for this? Why not just do the same thing we 
always do for injectable classes: If the customer wants a different
implementation, override the appropriate getter in their binder factory.